sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -309,6 +309,12 @@ object SQLConf {
     .stringConf
     .createOptional
 
+  val THRIFTSERVER_INCREMENTAL_COLLECT =
```

Comment:
I think this makes this option for "public"; I see some other options here marked as `.internal()`. I don't know whether this is meant to be further exposed. It might be more conservative to make it internal for the moment? but yes seems sensible to make a config key constant like this.